utils/build.ps1

```diff
@@ -1366,7 +1366,7 @@ function Get-Dependencies {
 
     # TODO(compnerd) stamp/validate that we need to re-extract
     New-Item -ItemType Directory -ErrorAction Ignore $BinaryCache\toolchains | Out-Null
-    Extract-Toolchain "$PinnedToolchain.exe" $BinaryCache $PinnedToolchain.TrimStart("swift-").TrimEnd("-a-windows10")
+    Extract-Toolchain "$PinnedToolchain.exe" $BinaryCache $PinnedToolchain.TrimStart("swift-").TrimEnd("-arm64").TrimEnd("-a-windows10")
```

Comment:
A regex wouldn't be bad. I've seen some oddities with the current implementation (using `installer.exe` locally gives `nstaller` for the name). I think that doing that subsequently would be a good balance - clean up the behaviour but unblock usage.